packages/api/package.json

```diff
@@ -5,15 +5,17 @@
   "scripts": {
     "test": "jest --config=jest.config.js",
     "lint": "eslint --ext .js,.ts,.json -c ../../.eslintrc.json .",
-    "lint:fix": "npm run lint -- --fix"
+    "lint:fix": "npm run lint -- --fix",
+    "start": "ts-node orm/index.ts",
+    "typeorm": "node --require ts-node/register ./node_modules/typeorm/cli.js"
```

Comment:
The `start` command here can be removed.  I simply missed this during my cleanup.
And while the `typeorm` script was useful to get familiar with the tool, I didn't use it a great deal, and it can probably be removed also.

Reply:
Removed in 7dd84c15da6cd3fd23c1a3aa51879a92a5d3b96d